src/ctia/http/routes/bundle.clj

```diff
@@ -318,7 +318,7 @@
 
 (defn bulk-params []
   {:refresh
-   (get-in @properties [:ctia :store :bundle-refresh] false)})
+   (get-in @properties [:ctia :store :bundle-refresh] "false")})
```

Comment:
hmmmm.... that feel bad to use a string instead of a bool. I though properties were coerced.

edit: ok remember now that it is not a boolean but a 3 possible values, perhaps it would be best to use Refresh as an enum of keywords instead of strings.

Reply:
yeah that hurts I agree :-)